server/routes/overview/overviewController.ts

```diff
@@ -3,17 +3,22 @@ import EducationAndTrainingView from './educationAndTrainingView'
 import OverviewView from './overviewView'
 import SupportNeedsView from './supportNeedsView'
 import { CuriousService } from '../../services'
+import EducationAndWorkPlanService from '../../services/educationAndWorkPlanService'
 
 export default class OverviewController {
-  constructor(private readonly curiousService: CuriousService) {}
+  constructor(
+    private readonly curiousService: CuriousService,
+    private readonly educationAndWorkPlanService: EducationAndWorkPlanService,
+  ) {}
 
   getOverviewView: RequestHandler = async (req, res, next): Promise<void> => {
     const { prisonNumber } = req.params
     req.session.createGoalForm = undefined
 
     const { prisonerSummary } = req.session
+    const actionPlan = await this.educationAndWorkPlanService.getActionPlan(prisonNumber, req.user.token)
```

Comment:
Excellent - we are using the user token for this API call 👍 Thats the correct one to use in this case 👍